weave/util.py

```diff
@@ -21,25 +37,36 @@ def init_sentry():
     except ImportError:
         return
     # Disable logs going to Sentry. Its slow
-    sentry_sdk.init(integrations=[LoggingIntegration(level=None, event_level=None)])
+    sentry_sdk.init(integrations=[LoggingIntegration(level=None, event_level=None)], before_send=before_send)
 
 
 def raise_exception_with_sentry_if_available(
```

Comment:
Let's completely remove `raise_exception_with_sentry_if_available`. This is now a no-op and should be removed from all callsites

Reply:
Alternatively (if you wanted to have less code change, then this function could be responsible for associating the fingerprint with the error)